docs/docs/platform/quickstarts/react.mdx

```diff
@@ -0,0 +1,602 @@
+---
+title: 'Quickstart: React'
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Quickstart: React
+
+## Introduction
+
+This quickstart guide provides the steps you need to build a simple React app
+powered by Nhost for the backend. It includes:
+
+- Database: [PostgreSQL](https://www.postgresql.org/)
+- Instant GraphQL API: [Hasura](https://hasura.io/)
+- Authentication: [Hasura Auth](https://github.com/nhost/hasura-auth/)
+- Storage: [Hasura Storage](https://hub.docker.com/r/nhost/hasura-storage)
+- [Nhost CLI](https://docs.nhost.io/reference/cli) for local development
+
+By the end of this guide, you'll have a full-stack app that allows users to log
+in to access a protected dashboard and update their profile information.
+
+## Prerequisites
+
+Before getting started, let’s make sure that your development environment is
+ready.
+
+- You’ll need **Node.js** version 14 or later.
+  [Install it from here](https://nodejs.org/en/).
+- You’ll need the **Nhost CLI** and **Docker**. View the
+  [installation instructions](https://docs.nhost.io/platform/overview/get-started-with-nhost-cli).
+- You’ll need a **Github repository**. Create a
+  [new repository](https://github.com/new).
+
+## Project setup
+
+### Create a new Nhost app
+
+First things first, we need to create a new Nhost project.
+
+So, log in to your Nhost dashboard and click the **Create your first app**
+button.
+
+![nhost-first-app](/img/quickstarts/create-app-step-1.png)
+
+Next, give your new Nhost app a name, select a geographic region for your Nhost
+services and click **Create App**.
+
+![nhost-first-app](/img/quickstarts/create-app-step-2.png)
+
+After a few seconds, you should get a PostgreSQL database, a GraphQL API with
+Hasura, file storage, and authentication already set up.
+
+:::info
+You can also connect your Nhost app to a GitHub repository. When you do this, any updates you push to your code will automatically be deployed. [Learn more](https://docs.nhost.io/platform/github-integration).
+:::
+
+## Initialize the app
+
+### Create a React app
+
+The simplest way to create a new React application is by using the tool called
+`create-react-app`, which bootstraps a React app for you without the hassle of
+configuring everything yourself.
+
+So, open your terminal, and run the following command:
+
+```bash
+npx create-react-app my-nhost-app --template quickstart-nhost
+```
+
+:::info
+This command uses an [existing template](https://github.com/gdangelo/cra-template-quickstart-nhost), through the `--template` flag, which already contains the React components and pages we'll use for this guide.
```

Comment:
The templates are from the GH account's author. We should ideally move them to our repo

Reply:
Yes. I'll create a separate issue for this.

Reply:
https://github.com/nhost/nhost/issues/523